### PR TITLE
Implement `accounts/id/{staking-info, balance-info, vesting-info}` & `pallets/staking/progress`

### PR DESCRIPTION
**Pull request by emostov**
Implement:
- `accounts/accountId/staking-info`
- `accounts/accountId/balance-info`
- `accounts/accountId/vesting-info`
- `pallets/staking/progress`

Relates to #227

### CODE REVIEW

File: src/controllers/pallets/PalletsStakingProgressController.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import { ApiPromise } from '@polkadot/api';
+import { RequestHandler } from 'express';
+
+import { PalletsStakingProgressService } from '../../services';
+import AbstractController from '../AbstractController';
+
+/**
+ * GET get progress on the general staking pallet system
```
Comment (joepetrowski) [inline src/controllers/pallets/PalletsStakingProgressController.ts:8]:
```suggestion
 * GET progress on the general Staking pallet system.
```

File: src/controllers/pallets/PalletsStakingProgressController.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+ *
+ * Paths:
+ * - (Optional) `number`: Block hash or height at which to query. If not provided, queries
+ *   finalized head.
```
Comment (joepetrowski) [inline src/controllers/pallets/PalletsStakingProgressController.ts:12]: This is a param now, not a path, right?

File: src/controllers/accounts/AccountsVestingInfoController.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import { AccountsVestingInfoService } from '../../services';
+import AbstractController from '../AbstractController';
+
+/**
+ * GET vesting information for an address.
+ *
+ * Paths:
+ * - `address`: Address to query.
+ * - (Optional) `number`: Block hash or height at which to query. If not provided, queries
+ *   finalized head.
```
Comment (joepetrowski) [inline src/controllers/accounts/AccountsVestingInfoController.ts:15]: Not a path, right?

Comment (emostov) [reply src/controllers/accounts/AccountsVestingInfoController.ts:15]: Yeah, forgot to update that part of the docs for this and the other place this happened

File: src/controllers/index.ts [hunk 4]
```diff
@@ -2,7 +2,7 @@ export * from './transaction';
 export * from './accounts';
 export * from './blocks';
 export * from './runtime';
-// export * from './pallets';
+export * from './pallets';
```
Comment (danforbes) [inline src/controllers/index.ts:5]: :tada:

**Review by danforbes — APPROVED** [review]

**Pull request merged**